### ISSUE TITLE
Support more SSL ciphers

### ISSUE THREAD

**Issue by dbellavista** [tomav/docker-mailserver#410]
The Dovecot SSL cipher suite is really strict and programs such as node.js with the default ciphers are unable to connect using TLS.

The only way to connect is to use `DES-CBC3-SHA` which is not secure (BEAST attack).

Those are the default nodejs ciphers:

```
ECDHE-RSA-AES128-GCM-SHA256:
ECDHE-ECDSA-AES128-GCM-SHA256:
ECDHE-RSA-AES256-GCM-SHA384:
ECDHE-ECDSA-AES256-GCM-SHA384:
DHE-RSA-AES128-GCM-SHA256:
ECDHE-RSA-AES128-SHA256:
DHE-RSA-AES128-SHA256:
ECDHE-RSA-AES256-SHA384:
DHE-RSA-AES256-SHA384:
ECDHE-RSA-AES256-SHA256:
DHE-RSA-AES256-SHA256:
HIGH:
!aNULL:
!eNULL:
!EXPORT:
!DES:
!RC4:
!MD5:
!PSK:
!SRP:
!CAMELLIA
```

**Comment by tomav** [comment]
Hi @dbellavista, which do you suggest?
Want to contribute?

**Comment by tomav** [comment]
They may be aligned to Postfix configuration:

```
tls_high_cipherlist=EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA384:EECDH+aRSA+SHA256:EECDH:+CAMELLIA256:+AES256:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:!ECDSA:CAMELLIA256-SHA:AES256-SHA:CAMELLIA128-SHA:AES128-SHA
```

**Comment by tronicum** [comment]
seems to be fixed with 2a15ac619e5d3de81c3ff4d6620c2a3a0f8e4f11

**Comment by tomav** [comment]
Yep.
